Sources/MongoSwift/ConnectionString.swift

```diff
@@ -40,6 +40,22 @@ internal class ConnectionString {
             }
         }
 
+        if let heartbeatFrequencyMS = options?.heartbeatFrequencyMS {
+            guard heartbeatFrequencyMS >= 500 && heartbeatFrequencyMS <= Int32.max else {
```

Comment:
500 is minHeartbeatFrequencyMS as defined in [SDAM](https://github.com/mongodb/specifications/blob/master/source/server-discovery-and-monitoring/server-monitoring.rst#minheartbeatfrequencyms).